contributing/ci-ome-files.txt

```diff
@@ -0,0 +1,181 @@
+OME Files jobs
+--------------
+
+All jobs are listed under the :jenkinsview:`Files` view tab of Jenkins.
+
+.. list-table::
+        :header-rows: 1
+
+        -       * Job task
+                * stable series
+                * development series (0.1.x)
```

Comment:
0.2.x ? or alternatively remove from the header to minimize the number of places to update.